tsconfig.json

```diff
@@ -59,9 +59,7 @@
     "src/analysis/retail/hunter/beastmastery",
     "src/analysis/retail/hunter/marksmanship",
     "src/analysis/retail/hunter/survival",
-    "src/analysis/retail/hunter/shared",
-    "src/analysis/retail/shaman/restoration",
-    "src/analysis/retail/shaman/shared"
+    "src/analysis/retail/hunter/shared"
```

Comment:
Mistaken removal of shaman from tsconfig?

Reply:
Saw that on another commit ( https://github.com/WoWAnalyzer/WoWAnalyzer/pull/6915/commits/a1588eea958555d7803f2aeb4ee9ff05f66cd676 ) and thought it was required as part of the reactivation of the spec. Doesn't look like it makes any difference however, so now I'm confused as to what exactly is "excluded" about these specs.

Reply:
You know that's a fair point I'm not sure what the deal is with this partial list of specs here. Still, it is not needed for reactivation and we shouldn't change it until we know why we're changing it.

Reply:
If you're re-enabling a spec, it should be removed from tsconfig. The exclusion only applies to un-imported files in dev mode but it seems to do weird things with lazy loading (?) in production if you enable a spec and don't remove from tsconfig. We'll see if there are any unusual errors with resto shaman 😅

Reply:
Oh shit my bad I think I got mixed up on what this was - I guess we'll see what happens lmao